qutebrowser/browser/commands.py

```diff
@@ -849,6 +850,11 @@ def tab_focus(self, index: {'type': (int, 'last')}=None, count=None):
         if index is None and count is None:
             self.tab_next()
             return
+        if index:
+            try:
+                index = int(index)
+            except ValueError:
+                raise cmdexc.CommandError("tab-focus takes an int or 'last' not: {}".format(index))
```

Comment:
Oh I suppose it would do the same thing. Since typing until there is only one option left then pressing enter passes a string through I was going to add a title to index lookup there for one less keypress but that seems like way too much effort. I'll change it back.